lib/twilio-ruby.rb

```diff
@@ -10,7 +10,7 @@
 require 'json'
 
 require 'twilio-ruby/version' unless defined?(Twilio::VERSION)
-require 'rack/twilio_webhook_authentication' if defined?(Rack)
+require 'rack/twilio_webhook_authentication' if defined?(Rack) && (Gem.loaded_specs["rack"].version > Gem::Version.new("2"))
```

Comment:
Think this can be made more specific if we include `Rack::MediaType` as the needed depdendency (since that's what's required in `twilio_webhook_authentication`). E.g.,
```
require 'rack/twilio_webhook_authentication' if defined?(Rack) && defined?(Rack::MediaType)
```

Reply:
Done.